test/specs/chrome.js

```diff
@@ -1,28 +1,28 @@
 import { describe } from '../utils/mochaw'
 import { supportedLanguages } from '../config.json'
-import { welcomeScenarios } from './scenarios/welcome'
-import { documentSelectorScenarios } from './scenarios/documentSelector'
-import { documentScenarios } from './scenarios/document'
+// import { welcomeScenarios } from './scenarios/welcome'
```

Comment:
Maybe you forgot to remove the comments in this file

Reply:
whoops yeah forgot to do that. Will remove double slashes

Reply:
fixed